src/common/Card.js

```diff
@@ -6,10 +6,12 @@ export default function Card({ children }) {
 }
 
 const CardBG = styled.div`
+  display: inline-block;
   background: white;
   border-radius: 12px;
   box-shadow: var(--primary-shadow);
   margin: 30px 0;
   padding: 15px;
   position: relative;
```

Comment:
Why do you use `position: relative;` here?

Reply:
I used it to position the buttons at the bottom, but it didn't work and I forgot to remove it here. Now I fixed the buttons so the `position:relative` can stay :)